src/main/resources/templates/db_scripts/schema.json

```diff
@@ -67,6 +67,16 @@
       "snippetPath": "migration_scripts/rename_progress_current_to_exported.sql",
       "fromModuleVersion": "mod-data-export-2.0.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "default_data/create_default_mapping_profile.sql",
```

Comment:
do we know if these scripts are executed in sequence? Please verify that

Reply:
![image](https://user-images.githubusercontent.com/60778653/88692389-ad8e1400-d106-11ea-998a-41d8148a0332.png
Execution process from top to bottom